stairway/src/main/java/bio/terra/stairway/FlightFilter.java

```diff
@@ -233,7 +354,96 @@ public String getParameterName() {
     public enum Datatype {
       STRING,
       TIMESTAMP,
+      LIST,
```

Comment:
What this type means isn't obvious to me. And the semantics in FlightFilterAccess that convert to JSON are also not obvious. Maybe some documentation about what this means?

I'm noticing that documentation about using the FlightFilter is only in the comment at the top of this module and is not comprehensive. Not your problem, but something for me to add to the Stairway list.